src/main/java/no/nav/foreldrepenger/regler/uttak/fastsetteperiode/grunnlag/Dødsdatoer.java

```diff
@@ -1,12 +1,14 @@
 package no.nav.foreldrepenger.regler.uttak.fastsetteperiode.grunnlag;
 
 import java.time.LocalDate;
+import java.util.Optional;
 
 public final class Dødsdatoer {
 
     private LocalDate søkersDødsdato;
     private LocalDate barnsDødsdato;
-    private boolean alleBarnDøde;
+
+    private Optional<Integer> gjenlevendeBarn;
```

Comment:
Hva er det vi trenger antall levende barn til? Ser ut for meg at vi bare trenger om alle barn er døde

Reply:
Se knekkpunktIdentifier.finnKnekkpunktVedDød